opentelemetry-stdout/src/trace/exporter.rs

```diff
@@ -22,106 +20,136 @@ impl fmt::Debug for SpanExporter {
+        }
+        let mut print_header = true;
+        for kv in &span.instrumentation_lib.attributes {
+            if print_header {
+                println!("\t\t  Scope Attributes:");
+                print_header = false;
+            }
+            println!("\t\t\t ->  {}: {:?}", kv.key, kv.value);
+        }
+        println!();
+        println!("\t Name: {:?}", &span.name);
+        println!("\t TraceId: {:?}", &span.span_context.trace_id());
+        println!("\t SpanId: {:?}", &span.span_context.span_id());
+        println!("\t ParentSpanId: {:?}", &span.parent_span_id);
+        println!("\t Kind: {:?}", &span.span_kind);
 
-impl fmt::Debug for SpanExporterBuilder {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.write_str("SpanExporterBuilder")
-    }
-}
+        let datetime: DateTime<Utc> = span.start_time.into();
+        println!(
+            "\t Start time: {}",
+            datetime.format("%Y-%m-%d %H:%M:%S%.6f")
+        );
+        let datetime: DateTime<Utc> = span.end_time.into();
+        println!("\t End time: {}", datetime.format("%Y-%m-%d %H:%M:%S%.6f"));
+        println!("\t Status: {:?}", &span.status);
 
-impl SpanExporterBuilder {
-    /// Set the writer that the exporter will write to
-    ///
-    /// # Examples
-    ///
-    /// ```
-    /// use opentelemetry_stdout::SpanExporterBuilder;
-    ///
-    /// let buffer = Vec::new(); // Any type that implements `Write`
-    /// let exporter = SpanExporterBuilder::default().with_writer(buffer).build();
-    /// ```
-    pub fn with_writer(mut self, writer: impl Write + Send + Sync + 'static) -> Self {
-        self.writer = Some(Box::new(writer));
-        self
-    }
+        let mut print_header = true;
+        for kv in span.attributes.iter() {
+            if print_header {
+                println!("\t Attributes:");
+                print_header = false;
+            }
+            println!("\t\t ->  {}: {:?}", kv.key, kv.value);
+        }
 
-    /// Set the encoder that this exporter will use
-    ///
-    /// # Examples
-    ///
-    /// ```
-    /// use opentelemetry_stdout::SpanExporterBuilder;
-    ///
-    /// let exporter = SpanExporterBuilder::default()
-    ///     // Can be any function that can write formatted data
-    ///     // serde ecosystem crates for example provide such functions
-    ///     .with_encoder(|writer, data| Ok(serde_json::to_writer_pretty(writer, &data).unwrap()))
-    ///     .build();
-    /// ```
-    pub fn with_encoder(
-        mut self,
-        writer: impl Fn(&mut dyn Write, SpanData) -> TraceResult<()> + Send + Sync + 'static,
-    ) -> Self {
-        self.encoder = Some(Box::new(writer));
-        self
-    }
+        print_header = true;
+        for event in span.events.iter() {
+            if print_header {
+                println!("\t Events:");
+                print_header = false;
+            }
+            println!("\t\t Name: {:?}", event.name);
```

Comment:
Why do we want to print debug info for `event.name` here?

Reply:
Also we can just use it variable inline `println!("format {local_variable} arguments");`